ethstore/src/json/bytes.rs

```diff
@@ -0,0 +1,58 @@
+use std::{ops, str};
+use serde::{Deserialize, Deserializer, Error, Serialize, Serializer};
+use rustc_serialize::hex::{ToHex, FromHex, FromHexError};
+
+#[derive(Debug, PartialEq)]
+pub struct Bytes(Vec<u8>);
```

Comment:
this is the third definition of this type afaict (`rpc`, `json`, and now here) -- is it possible to be more DRY?